examples/index.js

```diff
@@ -1,4 +1,4 @@
-
+import 'babel-polyfill'
```

Comment:
At first, I could not get the examples running in IE 11. Importing babel-polyfill at that location helped. I think it had to do with the loading order of react-dom and babel-polyfill, similar to what is described here: https://stackoverflow.com/a/40928047.

Now after rebuilding node_modules it works, seems to be solved with react-dom 15.5.6. I could dig further into this and reinstall the old version, but for now I reverted that commit.